src/javascript.js

```diff
@@ -284,10 +284,16 @@ console.log(arry[1])
 
 
 // what is the index of 2 ? : 
+//it is the index of (1)
 
 // ==> TASK TWO
 // => explain to me how forEach works 
-
+let arry1 = [1, 2, 3, 4 , 5, 6, 7, 8];
+arry1.forEach(( arr,element, index) => {
+    console.log("arr:", arr, "Index:", index, "element:", element);
+})
```

Comment:
==> TASK ONE
=> Explain to me how indexOf works 

use indexOf to show me what is the index of 2 here 

let arry = [1, 2, 3];

what is the index of 2 ? :